mhlib/app.go

```diff
@@ -177,31 +176,33 @@ func (a *App) apply(configFile string, simulate bool) (*[]interface{}, error) {
 
 	// "specify the exact chart version to install. If this is not specified, the latest version is installed"
 	if chartVersion != nil {
-		cmd = append(cmd, []interface{}{"--version", *chartVersion}...)
+		cmd = append(cmd, "--version")
+		cmd = append(cmd, *chartVersion)
 	}
 
 	if simulate {
 		// "enable verbose output"
-		cmd = append(cmd, []interface{}{"--debug"}...)
+		cmd = append(cmd, "--debug")
 
 		// "simulate an upgrade"
-		cmd = append(cmd, []interface{}{"--dry-run"}...)
+		cmd = append(cmd, "--dry-run")
 	}
 
 	// "force resource update through delete/recreate if needed"
-	cmd = append(cmd, []interface{}{"--force"}...)
+	cmd = append(cmd, "--force")
 
 	// "if a release by this name doesn't already exist, run an install"
-	cmd = append(cmd, []interface{}{"--install"}...)
+	cmd = append(cmd, "--install")
 
 	// "performs pods restart for the resource if applicable"
-	cmd = append(cmd, []interface{}{"--recreate-pods"}...)
+	cmd = append(cmd, "--recreate-pods")
 
 	// "namespace to install the release into. (Default: 'default')"
-	cmd = append(cmd, []interface{}{"--namespace", a.Namespace}...)
+	cmd = append(cmd, "--namespace", a.Namespace)
 
 	// Make `helm upgrade` read overrides from stdin
-	cmd = append(cmd, []interface{}{"--values", "-"}...)
+	cmd = append(cmd, "--values")
+	cmd = append(cmd, "-")
```

Comment:
Should line 201 be broken up to match the break up of 204 / 205?  `--values -` tells helm to take values from stdin.

Reply:
both are valid and will do the same, but 201 line is probably easier to read than splitting it in two lines like I did in 204/205 (so I updated the PR)

Given the `append` is variadic function, it is possible to chain the arguments
https://golang.org/pkg/builtin/#append